src/common/escape.h

```diff
@@ -50,13 +49,13 @@ void escape_json_attr(const char *buf, size_t src_len, char *out);
 //   std::cout << xml_stream_escaper(xml_input) << std::endl;
 
 struct xml_stream_escaper {
-  boost::string_view str;
+  std::string_view str;
```

Comment:
please 
```c++
#include <string_view>
```
for using `std::string_view`.

Reply:
this also applies to other header files where `<boost/utility/string_view.hpp>` is removed.

Reply:
done